src/components/ArticleManager.jsx

```diff
@@ -0,0 +1,58 @@
+// header
+// nav bar
+// article card
+// comment manager > comment card
+// votes
+
+import { useState } from "react";
+import { useParams } from "react-router-dom";
+import { useEffect } from "react";
+
+export default function ArticleManager() {
+  const [selectedArticle, setSelectedArticle] = useState();
+  const { article_id } = useParams();
+
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    fetch(`https://news-app-n80t.onrender.com/api/articles/${article_id}`)
+      .then((data) => {
+        return data.json();
+      })
+      .then((body) => {
+        setSelectedArticle(body);
+        setIsLoading(false);
+      })
+      .catch((err) => {
+        console.log(err);
```

Comment:
Rather than console logging the error (which is not useful to the user), implement an `isError` pattern: notes are here... [https://notes.northcoders.com/courses/js-front-end/error-handling](url)